stories/EventCard.stories.js

```diff
@@ -0,0 +1,115 @@
+import VsGrid from '@/components/grid/Container.vue';
+import VsRow from '@/components/grid/Row.vue';
+import VsCol from '@/components/grid/Col.vue';
+import VsEventCard from '@/components/event-card/EventCard.vue';
+import VsList from '@/components/list/List.vue';
+
+export default {
+    component: VsEventCard,
+    title: 'EventCard',
+    decorators: [() => ({
+        template: '<div style="max-width:900px;"><story /></div>',
+    })],
+};
+
+const Template = (args) => ({
+    components: {
+        VsEventCard,
+        VsList,
+        VsCol,
+        VsRow,
+        VsGrid,
+    },
+    setup() {
+        return {
+            args,
+        };
+    },
+    template: `
+        <VsEventCard v-bind="args">
+                <template v-slot:event-card-header> {{args.title}} </template>
+
+                <template v-slot:event-card-date> {{ args.dates }} </template>
+
+                <template v-slot:event-card-content>
+
+                    <p class="mb-025"> {{ args.description }} </p>
+
+                    <VsRow class="mb-025">
+                        <VsCol
+                            sm="12"
+                            md="4"
+                            
+                        >
+                            <dl class="mb-0">
+                                <div v-if="args.times">
+                                    <dt class="d-inline">Time: </dt>
+                                    <dd class="d-inline"> {{ args.times }}</dd>
+                                </div>
+                                    
+                                <div v-if="args.price">
+                                    <dt class="d-inline">Price: </dt>
+                                    <dd class="d-inline"> {{ args.price }}</dd>
+                                </div>
+                                    
+                                <div v-if="args.location">
+                                    <dt class="d-inline">Location: </dt>
+                                    <dd class="d-inline"> {{ args.location }}</dd>
+                                </div>                                
+                                <div v-if="!(args.registrationDeadline && args.contact)">
+                                    <dt class="d-inline">Organiser: </dt>
+                                    <dd class="d-inline m-0"> {{ args.organizer }}</dd>
+                                </div>
+                            </dl>
+                        </VsCol>
+                        <VsCol
+                            sm="12"
+                            md="5"
+                        >
+                            <dl class="m-md-0">
+                                <div v-if="args.registrationDeadline">
```

Comment:
It's not semantically correct to have a div as a child of dl, I think you should be able to replace the "div" with "template" which should hopefully do the same thing